src/mmw/apps/modeling/calcs.py

```diff
@@ -92,3 +92,42 @@ def point_source_pollution(geojson):
         'name': 'pointsource',
         'categories': point_source_results
     }
+
+
+def catchment_water_quality(geojson):
+    """
+    Given a GeoJSON shape, retrieve Catchment Water Quality data
+    from the `drb_catchment_water_quality` table to display
+    in the Analyze tab.
+
+    Returns a dictionary to append to the outgoing JSON for analysis
+    result
+    """
+    geom = GEOSGeometry(geojson, srid=4326)
+    table_name = 'drb_catchment_water_quality'
+    sql = '''
+    SELECT nord, areaha, tn_tot_kgy, tp_tot_kgy, tss_tot_kg,
```

Comment:
Consider indenting the sql statement here to match the point source sql's indentation.